docker/demisto-sdk/README.md

```diff
@@ -0,0 +1,54 @@
+# Demisto-SDK docker
+
+Run demisto-sdk commands from within a docker.
+
+You can use this image to run validations locally on your work environment or inside your ci.
+
+## The Content Repository
+
+To use the Demisto-SDK you should have a content-like repository (or use the official [content repository](https://github.com/demisto/content)).
+You can generate one from this [template](https://github.com/demisto/content-external-template)
+
+## Mounts
+
+We use volume mount in our environment to run on local content repository.
+Please note that there's a performance issue with mount on MacOS and Windows.
+You can solve the performance issue by running on Linux/Windows WSL2 or by cloning your repository to the `/content` directory inside the container.
+
+## Examples
+
+(All examples from the [content](https://github.com/demisto/content) repository).
+
+### Validate command
+
+```sh
+docker run -it --rm --mount type=bind,source="$(pwd)",target=/content demisto/demisto-sdk:<tag> demisto-sdk validate -i Packs/ipinfo/Integrations/ipinfo_v2
+```
+
+#### Break down the command
+
+- docker run  
+    Creates a container and run a command
+- -it  
+    Keep the stdin open and tty
```

Comment:
```suggestion
    Keeps the `stdin` open and tty
```

`and tty`?

Reply:
https://en.wikipedia.org/wiki/Tty_(Unix)